packages/markdown-it-container/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
No TS in this package, but our ESLint setup requires a TSConfig file to be present.